hack/operator-garden-seed-down.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+# SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Gardener contributors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+PATH_KIND_KUBECONFIG=""
+PATH_GARDEN_KUBECONFIG=""
+
+parse_flags() {
+  while test $# -gt 0; do
+    case "$1" in
+    --path-kind-kubeconfig)
+      shift; PATH_KIND_KUBECONFIG="$1"
+      ;;
+    --path-garden-kubeconfig)
+      shift; PATH_GARDEN_KUBECONFIG="$1"
+      ;;
+    esac
+
+    shift
+  done
+}
+
+parse_flags "$@"
+
+# delete stuff gradually in the right order, otherwise several dependencies will prevent the cleanup from succeeding
+kubectl --kubeconfig "$PATH_GARDEN_KUBECONFIG" delete seed \
+  local \
+  --ignore-not-found \
+  --wait \
+  --timeout 5m
+
+kubectl --kubeconfig "$PATH_GARDEN_KUBECONFIG" annotate project local garden confirmation.gardener.cloud/deletion=true
+skaffold -f=skaffold.yaml --kubeconfig "$PATH_KIND_KUBECONFIG" delete -m gardenlet -p operator
+
+# workaround for https://github.com/gardener/gardener/issues/5164
+kubectl --kubeconfig "$PATH_KIND_KUBECONFIG" delete ns \
+  seed-local \
+  --ignore-not-found
+
+# cleanup namespaces that don't get deleted automatically
+kubectl delete ns gardener-system-seed-lease --ignore-not-found
```

Comment:
Does this work at all (which kubeconfig is used)?
Anyways, this can be removed, no? You cleanup the `Garden` next, so the virtual cluster will be deleted, and with it, this namespace.

Reply:
Nope, I deleted the statement 😄